docs/api/schema.yml

```diff
@@ -2149,6 +2164,7 @@ components:
         assigned_qa:
           type: string
           readOnly: true
+          nullable: true
```

Comment:
@jerivas I'm not sure why the schema changed here?

Reply:
Looks like the DRF upgrade includes a fix for fields with `null=True` https://github.com/encode/django-rest-framework/pull/8116. I confirmed the affected fields should indeed be nullable according to our models